datahub-web-react/src/app/entity/shared/tabs/Dataset/Validations/utils.tsx

```diff
@@ -227,10 +227,10 @@ export const getAssertionsSummary = (assertions: Assertion[]): AssertionStatusSu
 //  */
 export const createAssertionGroups = (assertions: Array<Assertion>): AssertionGroup[] => {
     // Pre-sort the list of assertions based on which has been most recently executed.
-    assertions.sort(sortAssertions);
+    const newAssertions = [...assertions].sort(sortAssertions);
```

Comment:
sometime it throwing error because of the `modify an array that has been marked as read-only`